test/Sentry.Profiling.Tests/TraceLogProcessorTests.ProfileInfo_Serialization_Works.verified.txt

```diff
@@ -645,7 +645,7 @@
       {
         function: System.Reactive.Linq.Observable.Merge(class System.IObservable`1<!!0>[]),
         module: system.reactive,
-        in_app: true
+        in_app: false
```

Comment:
interesting that it was `true` before

Reply:
Yes, I think there was a bug in the logic we had previously to identify InApp frames.